enhancements/two-node-fencing/tnf.md

```diff
@@ -123,6 +127,26 @@ This functionality exists within RHEL-HA, but a wrapper will be provided to take
 
 When starting etcd, the OCF script will use etcd's cluster ID and version counter to determine whether the existing data directory can be reused, or must be erased before joining an active peer.
 
+
+### Summary of Changes
+
+At a glance, here are the components we are proposing to change:
+| Component                                                         | Change                                                                                                                                |
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [Feature Gates](#feature-gate-changes)                            | Add a new `DualReplicaTopology` feature to the `CustomNoUpgrade` feature set                                                          |
```

Comment:
might be nit picking, sorry, but this still isn't quite correct: the `CustomNoUpgrade` set does not consist of a fixed set of features (so you can't add a feature to it), but it allows to enable or disable any feature manually  (In contrast to the tech preview feature set, which consists of a fixed, immutable set of features).
Maybe "Add a new `DualReplicaTopology` feature, which can be enabled with the `CustomNoUpgrade` feature set"?

Reply:
third time is the charm!

Reply:
👍🏼